locations/mlk-nk.yml

```diff
@@ -53,19 +53,15 @@ networks:
     name: mesh_nno_5
     prefix: 10.31.69.33/32
     ipv6_subprefix: -11
-    mesh_metric: 1024
     mesh_metric_lqm: ['default 0.6']
-    ptp: true
```

Comment:
why deleting this?

Reply:
The device is configured in AP mode therefore it can be ptmp.